docs/README.md

```diff
@@ -101,6 +101,8 @@ There are many global variables defined in the <a href="{{ config.repo_raw_url }
     - edit_uri: {{ config.edit_uri }}
     - ks_branch: {{ config.ks_branch }}
     - ks_tag: {{ config.ks_tag }}
+    - ks_latest_regular_release: {{ config.ks_latest_regular_release }}
+    - ks_latest_release: {{ config.ks_latest_release }}
```

Comment:
I suggest we converge to only use one latest (of a regular release) .   All references except one are using the latest of regular release in any case.

Reply:
No, most of the uses are for the self-references that need to be to the release being made regardless of what kind of release it is.

Reply:
@MikeSpreitzer (@eaepstein @pdettori ) I don't agree.  I think that at this stage of the project (it may be changed in the future) we want users to use only "regular release"  . The other releases are point releases aimed for developers for specific task (test a fix, test a new feature, etc..) 
As our web-site is aimed to users - all references to 1latest1 on the web site should refer to the latest `regular release`.   
The few developers who use/test the RCs don't need to use direct references for theses RCs on our web documentation in order to use it. 
BTW, I believe that most of the team actually supported to even  generate the docs only for `regular releases`  , and not for other intermediate releases, but we didn't vote on that and just let it continue :(

Reply:
>The other releases are point releases aimed for developers for specific task (test a fix, test a new feature, etc..)

Does this translate to mean 0.23.0 is a "regular" release but 0.23.1 is not? And therefor 0.23.1 cannot be labeled as "latest" ?

Reply:
@ezrasilvera:

> @MikeSpreitzer (@eaepstein @pdettori ) I don't agree. I think that at this stage of the project (it may be changed in the future) we want users to use only "regular release" .

I agree with that.

The problem is all the self-references that have to be updated when making a release of any kind, including a test (candidate) release. See https://github.com/kubestellar/kubestellar/blob/v0.23.0/docs/content/direct/release.md#making-a-new-kubestellar-release . Most of those self-references are in doc files. I introduced `config.ks_latest_release` to be used in place of those self-references, so that making a release of any kind (including a test release) requires editing only the definition of `config.ks_latest_release` instead of all the places where it appears in the docs.

Reply:
@eaepstein: I think that when @ezrasilvera wrote about "point releases" he did not mean "point" in the technical semver sense of major.minor.patch; rather, he meant "point" in the sense of "intended for a narrow purpose". A release identified simply as "major.minor.patch" _is_ a regular release, not a test (candidate) release or alpha or beta or other unusual thing.

Reply:
@ezrasilvera: can we separate a change to our policy of which releases appear on the website from this PR?